RPGSystem/DiceTool.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace RPGSystem
+{
+    public partial class DiceTool : Form
+    {
+        public DiceTool()
+        {
+            InitializeComponent();
+        }
+
+        private void BtnThrewADice_Click(object sender, EventArgs e)
+        {
+            RandomNumberGenerator randomNumberGenerator = new RandomNumberGenerator();
```

Comment:
I would recommend making randomNumberGenerator private class field and initialize it inline to not to create a new instance every time a user clicks on a throw button.

Reply:
Fixed